src/Core/Unix/CoreUnix.cpp

```diff
@@ -294,8 +294,8 @@ namespace VeraCrypt
 		if (FilesystemPath ("/media").IsDirectory())
 			return "/media/veracrypt";
 
-		if (FilesystemPath ("/mnt").IsDirectory())
-			return "/mnt/veracrypt";
+		if (FilesystemPath ("/run/media").IsDirectory())
+			return "/run/media/veracrypt"
```

Comment:
This won't compile without the semicolon:
```suggestion
			return "/run/media/veracrypt";
```

While I think /run/media is the right modern location for automatically mounted devices, I think /mnt should still be kept as a fallback. The reason is that placing the mountpoint into /tmp should be avoided, and there are still some old systems (such as Slackware) which veracrypt is ported over to which don't have /media nor /run in the file hierarchy as they still use /var for it.